fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -63,6 +63,14 @@ struct ProxyStats {
 	Counter conflictRanges;
 	Counter keyServerLocationIn, keyServerLocationOut, keyServerLocationErrors;
 	Version lastCommitVersionAssigned;
+	double transactionRateAllowed, batchTransactionRateAllowed;
```

Comment:
We should initialize the new fields added if we aren't already.

Reply:
Done.